zerver/tests/test_message_edit.py

```diff
@@ -1353,9 +1353,9 @@ def assert_is_topic_muted(
             users_to_be_notified_via_muted_topics_event.append(user_topic.user_profile_id)
 
         change_all_topic_name = "Topic 1 edited"
-        # This code path adds 17 (10 + 4/visibility_policy + 1/user + 1) to
+        # This code path adds 20 (13 + 4/visibility_policy + 1/user + 1) to
```

Comment:
To be honest, I think this comment is not really undestandable. I am not sure what "This code path" means here and what queries does it add to moving topic code. I think all the database work for moving a topic happens in `check_update_message` which is tested here and this itself is the function which is called from the views function. 

I remembered trying to look at the commit history for this line, but did not came to a conclusion then. Would try to look at it once again.

Reply:
I think either way works for me.